src/__tests__/render.js

```diff
@@ -113,3 +113,44 @@ test('renders options.wrapper around node', () => {
 </div>
 `)
 })
+
+describe('baseElement', () => {
```

Comment:
I really prefer to not use any nesting. This is all for just one test, please instead move the contents of beforeAll and afterAll to the inside of the test and use the `test` global rather than the `describe` and `it` globals.

Reply:
This breaks two other test that assume an empty body. The nesting is for isolation. What is the problem with that? The alternative is to create a new test which is worse IMO.

Reply:
I don't like the direction that nesting puts us in. It's a slippery slope IMO. Maybe I'm just too scarred by past testbases. I'm much more willing to create a new test file than nesting tests.

Another thing we could do is just take the `beforeAll` and `afterAll` and put them inside the test. That would avoid the problems altogether. Could you do that please?